src/connection.rs

```diff
@@ -5,24 +5,9 @@ use std::rc::Rc;
 
 use mio::tcp::TcpStream;
 use mio::Timeout as TimeoutHandle;
-use mio::{
```

Comment:
Yeah. it makes it really hard to enumerate them visually. The [style guide](http://doc.rust-lang.org/nightly/style/style/imports.html) doesn't seem to offer much guidance in this though.